README.md

```diff
@@ -79,31 +79,40 @@ Below describes how Beam applications can be run directly on Nemo.
 
 ### Examples
 ```bash
+## Beam WordCount (from Beam website) example
+$ ./bin/run_beam.sh \
+    -job_id beam_wordcount \
+    -optimization_policy org.apache.nemo.compiler.optimizer.policy.DefaultPolicy \
+    -user_main org.apache.nemo.examples.beam.BeamWordCount \
+    -user_args "--runner=NemoRunner --inputFile=`pwd`/examples/resources/inputs/test_input_wordcount --output=`pwd`/outputs/wordcount"
+$ less `pwd`/outputs/wordcount*
+
 ## MapReduce example
```

Comment:
This is another WordCount example, right?
If so, please note the difference with the Beam's one.

Reply:
This one actually is the actual example from Beam, and the one below is our implementation of wordcount, which is a little different.

Reply:
Oh, what I meant was, please note it as a comment.

Reply:
You mean in the README?

Reply:
Oh OK. I got what you're saying.

Reply:
It's been reflected, sorry for the delay.